sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql

```diff
@@ -1353,7 +1337,9 @@ aggregates AS (
       STRUCT(ARRAY_CONCAT_AGG(browser_search_adclicks_urlbar_persisted)),
       STRUCT(ARRAY_CONCAT_AGG(scalar_parent_sidebar_opened)),
       STRUCT(ARRAY_CONCAT_AGG(scalar_parent_sidebar_search)),
-      STRUCT(ARRAY_CONCAT_AGG(scalar_parent_sidebar_link))
+      STRUCT(ARRAY_CONCAT_AGG(scalar_parent_sidebar_link)),
+      STRUCT(ARRAY_CONCAT_AGG(browser_ad_clicks_combined)),
+      STRUCT(ARRAY_CONCAT_AGG(browser_search_with_ads_combined))
```

Comment:
nit: I would switch the order here, to keep it consistent with the old versions (`browser_search_with_ads_combined`, then `browser_ad_clicks_combined`)